lib/blockscore/base.rb

```diff
@@ -53,22 +52,22 @@ def self.resource
       @resource ||= Util.to_underscore(to_s.split('::').last)
     end
 
-    def self.api_url
-      'https://api.blockscore.com/'
-    end
-
     def self.endpoint
       fail NotImplementedError, ABSTRACT_WARNING if equal?(Base)
 
-      "#{api_url}#{Util.to_plural(resource)}"
+      API_URL + Util.to_plural(resource)
     end
 
     def persisted?
-      !id.nil? && !attributes[:deleted]
+      !id.nil? && !attributes.fetch(:deleted, false)
```

Comment:
@mbj I thought about that, but I find `!(id.nil? || deleted?)` to be more difficult to understand.

The original example I read as "id is not nil and it's not deleted", while I felt like I was performing a few extra brain cycles to understand the second version.